starter/docs/commands/root_completion.md

```diff
@@ -0,0 +1,26 @@
+## root completion
+
```

Comment:
do we have completion package in starter or in core till now?

Reply:
Yes, if you run the starter CLI you'll notice a command for the same over there.
BTW these docs were autogenerated by Cobra :)